templates/graphql-js-starter/src/context.ts

```diff
@@ -1,9 +1,10 @@
 import express from 'express'
 import { RedisPubSub } from 'graphql-redis-subscriptions';
+import { PubSub } from 'graphql-subscriptions';
 import knex from 'knex'
 
 export interface GraphQLContext {
-  pubsub: RedisPubSub
+  pubsub: RedisPubSub | PubSub
```

Comment:
I would sacrifice strong typing to remove RedisSubscriptions from it. 
We can also use conditionals from TypeSctipt that will check if package exist

Reply:
Remove pubsub from here?

Reply:
RedisPubSUb is specific. We can simply use interface ?